online_store/gt_sport_v2.js

```diff
@@ -110,13 +117,13 @@ function displayContentInCart() {
 displayContentInCart();
 
 function calculateTotalPrice() {
-  let totalPrice=0;
-  for (let i=0; i < shoppingCart.length; i++){
+  let totalPrice = 0;
+  for (let i = 0; i < shoppingCart.length; i++) {
     totalPrice += shoppingCart[i].price;
   };
   alert(totalPrice);
 };
 
-shoppingCartEl.addEventListener(displayContentInCart);
+shoppingCartEl.addEventListener("click", displayContentInCart);
```

Comment:
It's not shoppingCartEl - because it's not a button, I think you want to use button with id='intoSC'.

Reply:
changed.